pkg/pillar/cmd/baseosmgr/baseosmgr.go

```diff
@@ -28,6 +28,11 @@ const (
 	// Time limits for event loop handlers
 	errorTime   = 3 * time.Minute
 	warningTime = 40 * time.Second
+
+	// last value of baseOsMgrContext.currentUpdateRetry for persistence
+	currentRetryUpdateCounterFile = types.PersistStatusDir + "/current_retry_update_counter"
+	// last value of baseOsMgrContext.configUpdateRetry for persistence
+	cfgRetryUpdateCounterFile = types.PersistStatusDir + "/cfg_retry_update_counter"
```

Comment:
You "spell"/name this "cfg" here and "config" everywhere else. I suggest "config" across the board to make it easier to find.

Reply:
Sorry, my mistake from previous versions